app/Models/Permission.php

```diff
@@ -42,22 +50,35 @@ class Permission extends Model
      */
     protected $guarded = ['id', 'created_at', 'updated_at'];
 
-     /**
-      * Cast values to correct type.
-      *
-      * @var array
-      */
-     protected $casts = [
-         'user_id' => 'integer',
-         'server_id' => 'integer',
-     ];
+    /**
+     * Cast values to correct type.
+     *
+     * @var array
+     */
+    protected $casts = [
+        'subuser_id' => 'integer',
+    ];
 
+    /**
+     * Find permission by permission node.
```

Comment:
Well there is a similar method below where it's called filter. Also there could be the same node in different servers right? So it actually does filter, not find

If you don't need an instance, why create one to call the method?